binancial/data/get_klines_realtime.py

```
@@ -48,30 +71,57 @@ def handle_socket_message(msg):
+            stream.start()
+            
+            # Store socket id for proper cleanup
+            socket_id = stream.start_kline_socket(
+                callback=handle_socket_message,
+                symbol=symbol,
+                interval=interval
+            )
+            
+            stream.join()
+            
+    except IOError as io_error:
```

Ruff (0.8.2)

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)